src/main/config/config.c

```diff
@@ -723,44 +725,54 @@ void activateConfig(void)
 #endif
 }
 
-void validateAndFixConfig(void)
+static void featureValidateSet(uint32_t mask)
```

Comment:
Not sure about the name, perhaps featureEnabledSet(), featureRequestedSet() instead of featureSet.